Dockerfiles/agent/README.md

```diff
@@ -76,16 +76,22 @@ DD_AC_INCLUDE = "image:cp-kafka image:k8szk"
 
 Please note that the `docker.containers.running`, `.stopped`, `.running.total` and `.stopped.total` metrics are not affected by these settings and always count all containers. This does not affect your per-container billing.
 
+### Datadog Cluster Agent
+
+As with Agent v6.2, you can use the [Datacong Cluster Agent](#https://github.com/DataDog/datadog-agent/blob/master/docs/cluster-agent/README.md).
```

Comment:
Let's bump that to 6.4.0 shall we?  We'll have the beta for 6.3.1 but 6.2.0 seems too early a start.